tools/config/mockery.yaml

```diff
@@ -18,3 +18,10 @@ packages:
           dir: "./auth/mocks"
           mockname: "PolicyServiceClient"
           filename: "policy_client.go"
+  github.com/absmach/magistrala/certs/pki/am-certs:
```

Comment:
why did you put this here?

Reply:
Its a much better way to generate mocks compared to `go generate`. I have used this because I could easily select the folder where the mock was being saved.